cafebabel/tests/test_articles.py

```diff
@@ -284,3 +284,35 @@ def test_article_to_translate_should_have_only_other_links(
     article.modify(language=language)
     response = client.get(f'/article/to-translate/?from=en&to=fr')
     assert 'Translate in ' not in response
+
+
+def test_article_with_tag(app, tag, article):
+    Article.objects(id=article.id).update_one(push__tags=tag)
+    assert Article.objects(tags__in=[tag]).count() == 1
+    article2 = Article.objects(tags__in=[tag]).first()
+    assert article2.tags[0].summary == 'summary text'
+    assert tag.slug == 'a-tag'
+    assert str(tag) == 'A tag (en)'
```

Comment:
Feels like these 2 asserts testing the slug value and the stringification should be in a different test.